### PR TITLE
Ember Syntax Theme and Ember Grammar

### PR DESCRIPTION
**Pull request by EWhite613**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2015 <Your name here>
```
Comment (sglanzer-fire) [inline LICENSE.md]: "Your name here" lol - you can copy the license from one of the other blueplanet projects

File: README.md [hunk 2]
```diff
@@ -1 +1,14 @@
...
+## Usage
+Change syntax theme to Atom Ember Frost
+
+## Theme:
+* Handlebars:
+	* ![Handlebars](images/handlebars-1.png)
+	* ![Handlebars](images/handlebars-2.png)
+
+## Notes:
+Be sure to download a Babel package that handles grammar for a good Ember-Atom experience
```
Comment (sglanzer-fire) [inline README.md:14]: grammar for javascript files?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "atom-ember-frost-syntax",
+  "theme": "syntax",
+  "version": "0.0.0",
+  "description": "A short description of your syntax theme",
```
Comment (sglanzer-fire) [inline package.json]: Description?

**Comment by sglanzer-fire** [comment]
👍

File: README.md (hunk 2 above)
Comment (EWhite613) [inline README.md:14]: @sglanzer Gonna rely on a good Babel package to handle a users javascript files atom editor css. Grammers are basically regex that give portions of the text the proper css class

Comment (sglanzer-fire) [inline README.md:14]: Yeah, this was more a comment about clarifying that the grammar was to handle the ES6 format javascript to pair with the hbs grammar

Comment (sglanzer-fire) [inline README.md:14]: Also, probably would be a good idea to suggest one here

**Pull request merged**